book/06-github/sections/2-contributing.asc

```diff
@@ -1,78 +1,78 @@
-This opens up a discussion thread with code review, and the owner and the contributor can then communicate about the change until the owner is happy with it, at which point the owner can merge it in.
+Sa paraang ito, hindi na kailangan mag-alala ng mga proyekto tungkol sa pagdagdag ng mga user bilang mga tagatulong na bigyan sila ng access sa pag-push.
+Maaari mag-force ang mga tao sa isang proyekto, mag-push dito, at mag-ambag sa kanilang mga pagbabago pabalik sa kanilang orihinal na repositoryo sa pamamagitan ng paglilikha ng tinatawag na Kahilingan ng Pull, na ating itatalakay sa susunod.
+Ito ay nagbubukas ng isang thread ng diskusyon sa pagsusuri ng code, at ang may-ari at nag-aambag ay maaari makikipag-usap tungkol sa pagbabago hanggang ang may-ari ay masaya dito, kung saan ang may-ari ay maaaring pagsamahin ito.
 
-To fork a project, visit the project page and click the ``Fork'' button at the top-right of the page.
+Para ma-fork ang isang proyekto, bisitahin ang pahina ng proyekto at i-click ang pindutan na ``Fork'' na nasa kanang itaas ng pahina.
 
-.The ``Fork'' button.
-image::images/forkbutton.png[The ``Fork'' button.]
+.Ang ``Fork'' na pindutan.
+image::images/forkbutton.png[Ang ``Fork'' na pindutan.]
 
-After a few seconds, you'll be taken to your new project page, with your own writeable copy of the code.
+Pagkatapos ng ilang segundo, dadalhin ka sa pahina ng iyong bagong proyekto, na may sariling kopya ng code na maaaring mabago.
 
 
 [[_github_flow]]
-==== The GitHub Flow
+==== Ang Daloy ng GitHub
 
 (((GitHub, Flow)))
-GitHub is designed around a particular collaboration workflow, centered on Pull Requests.
-This flow works whether you're collaborating with a tightly-knit team in a single shared repository, or a globally-distributed company or network of strangers contributing to a project through dozens of forks.
-It is centered on the <<_git_branching#_topic_branch>> workflow covered in <<_git_branching#_git_branching>>.
+Ang GitHub ay dinisenyo sa paligid ng isang partikular na workflow sa pakikipagtulungan, nakasentro sa mga Kahilingan ng Pull.
+Ang daloy na ito ay gumagana kung nakikipagtulungan ka sa isang mahigpit na pangkat sa isang solong ibinahaging repositoryo, o isang kompanyang ibinahagi sa mundo o network ng mga estranghero na nag-aambag sa isang proyekto sa pamamagitan ng dose-dosenang mga fork.
+Ito ay nakasentro sa workflow na <<_git_branching#_topic_branch>> na tinalakay sa <<_git_branching#_git_branching>>.
 
-Here's how it generally works:
+Narito kung paano ito gumagana:
 
-1. Fork the project
-2. Create a topic branch from `master`.
-3. Make some commits to improve the project.
-4. Push this branch to your GitHub project.
-5. Open a Pull Request on GitHub.
-6. Discuss, and optionally continue committing.
-7. The project owner merges or closes the Pull Request.
+1. I-fork ang proyekto
```

Comment:
`Magbukas ng isang Kahilingan ng Pull sa GitHub.` to `Magbukas ng isang Pull na Kahilingan sa GitHub.`